### ISSUE TITLE
"\Drupal calls should be avoided in classes, use dependency injection instead" is reported on Entity classes

### ISSUE THREAD

**Issue by pcambra** [mglaman/phpstan-drupal#12]
I think it shouldn't as per https://www.drupal.org/project/drupal/issues/2913224

How can I help to add this exception? :)

**Comment by mglaman** [comment]
@pcambra see https://github.com/mglaman/phpstan-drupal/blob/master/src/Rules/Drupal/GlobalDrupalDependencyInjectionRule.php#L33 :)

We just need to add the fully qualified class name to this list

```
        $whitelist = [
            // Typed data objects cannot use dependency injection.
            'Drupal\Core\TypedData\TypedDataInterface',
            // Render elements cannot use dependency injection.
            'Drupal\Core\Render\Element\ElementInterface',
            'Drupal\Core\Render\Element\FormElementInterface',
            'Drupal\config_translation\FormElement\ElementInterface',
        ];
```

**Comment by mglaman** [comment]
Merged, thanks @pcambra !
